src/js/components/CheckBox/StyledCheckBox.js

```diff
@@ -14,6 +14,11 @@ const hoverStyle = css`
     border-color: ${props =>
```

Comment:
Since the check got removed on line 42, we will need to check for it here somehow. I implemented some checks like this on this PR at line 128 of StyledButton.js https://github.com/grommet/grommet/pull/3877/files

Reply:
Is this comment still relevant @taysea ?